libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPAuthCodeHelper.kt

```diff
@@ -212,6 +220,18 @@ internal class IDPAuthCodeHelper private constructor(
                 false
             }
         }
+
+        override fun onPageFinished(webView: WebView?, url: String?) {
+            super.onPageFinished(webView, url)
+            if (webView != null) {
+                (webView.parent as? RelativeLayout)?.let { parentView ->
+                    parentView.findViewById<ProgressBar>(R.id.sf__loading_spinner)?.let { loadingText ->
+                        loadingText.visibility = View.INVISIBLE
```

Comment:
Showing spinner until web view loads, then hiding spinner and making web view visible.